neoexchange/core/management/commands/lightcurve_extraction.py

```diff
@@ -474,18 +479,16 @@ def handle(self, *args, **options):
                         movie_file = make_gif(frames_list, sort=False, init_fr=100, center=3, out_path=out_path, plot_source=True,
                                               target_data=frame_data, show_reticle=True, progress=True)
                         if "WARNING" not in movie_file:
+                            # Create DataProduct
+                            save_dataproduct(obj=block, filepath=movie_file, filetype=DataProduct.FRAME_GIF)
                             output_file_list.append('{},{}'.format(movie_file, data_path.lstrip(out_path)))
                             self.stdout.write("New gif created: {}".format(movie_file))
                         else:
                             self.stdout.write(movie_file)
-        alcdef_file.close()
+        save_dataproduct(obj=block.body, filepath=None, filetype=DataProduct.ALCDEF_TXT, filename=alcdef_filename, content=alcdef_txt)
```

Comment:
I'm getting a bit of disconnect between the `product` and the location of the actual file. 
Hopefully you can help me with this @zemogle?
`product` results in `products/65717_sin_20210112_1127860_ALCDEF_mafkCzh.txt` for the data product, but I cannot find any actual data product anywhere. Certainly not with the random string.

Reply:
This is inside your `MEDIA_ROOT` if you are using local file storage. If you're using `USE_S3` then its in there.